catroid/src/main/res/values/strings.xml

```diff
@@ -79,6 +79,16 @@
         You may also report the project on its page on the Catrobat community platform in case
         you are sure that it is harmful.</string>
 
+    <!-- Replace Api dialog -->
+    <string name="api_replacement_dialog_neutral">Upload with Keys anyway</string>
+    <string name="api_replacement_dialog_accept">Upload with Keys replaced</string>
+    <string name="api_replacement_dialog_warning">The project contains bricks that access the
+        web, and also seems to contain API keys. Please see our <a
+            href="https://wiki.catrobat.org/bin/view/Documentation/Web%20requests/">wiki</a> why uploading these keys can be dangerous
+        .</string>
+    <!-- -->
```

Comment:
test are falling cause of missing string resource `R.string.dialog_privacy_policy_text`
`22:57:24  /home/catroid/jenkins_slave/workspace/Catroid_PR-3873/catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/dialog/ReplaceApiKeyDialogTest.java:90: error: cannot find symbol
22:57:24  						UiTestUtils.getResourcesString(R.string.dialog_privacy_policy_text)`
did you forget to add it?